practical/groovy/open_omero_image_using_bf.groovy

```diff
@@ -0,0 +1,99 @@
+/*
+ * -----------------------------------------------------------------------------
+ *  Copyright (C) 2018 University of Dundee. All rights reserved.
+ *
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * ------------------------------------------------------------------------------
+ */
+
+/*
+ * This Groovy script shows how to open an OMERO Image in ImageJ
+ * using Bio-Formats importer.
```

Comment:
This script seems not to use Bio-Formats importer. Only the https://github.com/ome/training-scripts/pull/31/files#diff-66c94234b3208d29f85ff39a40086e75 does. Possibly the header was just copied from there ?

Reply:
Also, the name of this script suggests it opens the file using BF. But I think it is misnamed. Instead, the https://github.com/ome/training-scripts/pull/31/files#diff-66c94234b3208d29f85ff39a40086e75 does use BF, but it does not have it in its name.

Reply:
Aha, I was wrong, this script also uses BF importer. But it does not download the file into tmp directory I guess.

Reply:
It is hard to make a difference between the two opening scripts just based on the naming  ``using_bf`` here, because the other script is also using BF importer, but run as a command in IJ and after you downloaded the image locally first. I think the scripts should be renamed to capture this better.